scripts/test-runner.js

```diff
@@ -266,7 +266,7 @@ function runTest(name, code, options: PrepackOptions, args) {
   let delayUnsupportedRequires = code.includes("// delay unsupported requires");
   if (code.includes("// inline expressions")) options.inlineExpressions = true;
   if (code.includes("// do not inline expressions")) options.inlineExpressions = false;
-  if (code.includes("// omit invariants") || args.verbose) options.omitInvariants = true;
+  options.invariantLevel = code.includes("// omit invariants") || args.verbose ? 0 : 4;
```

Comment:
Why 4?

Reply:
4 roughly corresponds to the old invariant checking default